.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,11 +6,10 @@
 
 #### How to test / what to look for?
 
--
+1.
 
 #### Notes
 
-Resolves #
```

Comment:
I personally like this keyword-based linking. I know it's technically not necessary when branch was linked with the issue via the GitHub UI, but I think the keyword still helps when working mostly in the shell and communicates directly this relationship. I'm still using it.

Reply:
Don't get me wrong, for consistency sake I am ready to give it up though.

Reply:
I guess it is the other way around for me. I do not use it anymore, since the PR's are auto-linked. If it helps in your workflow we can keep it.

Reply:
That what I meant. If we aren't consistent than it does matter. And Lukas didn't use the keyword because you told him to. ;)

Reply:
True. In this case I meant Lumo, but whatever. 🙂 Lets keep it this way then.